MorozovVladyslav/HW7/3.py

```diff
@@ -0,0 +1,9 @@
+def number_of_characters(word):
```

Comment:
good job
but look at this implementation
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result